shared/lib/ortcustomops.cc

```diff
@@ -97,28 +98,31 @@ extern "C" ORTX_EXPORT OrtStatus* ORT_API_CALL RegisterCustomOps(OrtSessionOptio
   static std::vector<FxLoadCustomOpFactory> c_factories = {
     LoadCustomOpClasses<CustomOpClassBegin>
 #if defined(ENABLE_TF_STRING)
-    , LoadCustomOpClasses_Text
-#endif // ENABLE_TF_STRING
+    ,
```

Comment:
Any reason that the comma and the class name are separated into two lines? The existing one looks like more compact.

Reply:
clang-format did that. Are we not using the same coding conventions and standards as for ORT?

Reply:
could we instead do something like:
```cpp
  static std::vector<FxLoadCustomOpFactory> c_factories = {
    LoadCustomOpClasses<CustomOpClassBegin>,
#if defined(ENABLE_TF_STRING)
    LoadCustomOpClasses_Text,
#endif  // ENABLE_TF_STRING
    ...,
  };
```

Reply:
I remember if all of operators are disabled, the last comma after <CustomOpClassBegin> fails the build.  Maybe we can add <CustomOpClassEnd> to solve that.

Reply:
One alternative would be to change the factory setup so the factory has a method to return the shared_ptr instances for its enabled ops so we're not using a template unpack to expand the complete set of class names here. 

```c++
std::vector<std::shared_ptr<OrtCustomOp>> GetCustomOps() {
  std::vector<std::shared_ptr<OrtCustomOp>> ops;

#ifdef ENABLE_1
  ops.push_back(std::make_shared<CustomOp1>());
#endif

#ifdef ENABLE_2
  ops.push_back(std::make_shared<CustomOp2>());
#endif

  return ops;
}
```

Reply:
In the old ort C++ header files, some methods of CustomOp are non-const, so the class has to be created dynamically. But now, with some fixing, the custom op object can be a constant object.  We may solve this problem in by the c++ initializer list, and we may do it later.